internal/providers/terraform/aws/testdata/kinesis_firehose_delivery_stream_test/kinesis_firehose_delivery_stream_test.golden

```diff
@@ -0,0 +1,48 @@
+
+ Name                                                         Monthly Qty  Unit                    Monthly Cost 
+                                                                                                                
+ aws_elasticsearch_domain.test_cluster                                                                          
+ └─ Instance (on-demand, m4.large.elasticsearch)                      730  hours                        $134.32 
+                                                                                                                
+ aws_kinesis_firehose_delivery_stream.EnabledFalse                                                              
```

Comment:
We should update one of these test cases to have > 6PB of data ingested since that is the "weird" case.